### PR TITLE
fix(status-change-ui): fixed status overflow and busy icon size

### PR DESCRIPTION
**Pull request by ThomBos**
### What this PR does 📖
- 

### Which issue(s) this PR fixes 🔨
- Resolve #

### CODE REVIEW

**Review by stavares843 — APPROVED** [review]

**Comment by molimauro** [comment]
If we change that here we should change also on the UiUserState accordingly

**Review by molimauro — DISMISSED** [review]
fix also on the main user state component

**Comment by josephmcg** [comment]
those css classes didnt exist. plus we don't need to define these classes because there is a utility class.

name was properly overflowing with the current styles, just copy that. Please ask questions if anything is unclear, don't just switch labels

File: components/ui/User/State/State.vue [hunk 1]
```diff
@@ -75,9 +75,7 @@ export default Vue.extend({
       if (this.isTyping) {
         return 'typing'
       }
-      return this.user.did === iridium.id
-        ? 'online'
-        : this.users.ephemeral.status?.[this.user.did] ?? 'offline'
+      return 'busy'
```
Comment (molimauro) [inline components/ui/User/State/State.vue]: revert i guess

Comment (ThomBos) [reply components/ui/User/State/State.vue]: yep, done! that was just for testing it

**Pull request merged**